lib/ffmpeg/encoderbase.py

```diff
@@ -89,7 +90,8 @@ def gen_output_opts(self):
 
     # WA: LDB is not enabled by default for HEVCe on gen11+, yet.
     if get_media()._get_gpu_gen() >= 11 and self.codec.startswith("hevc"):
-      opts += " -b_strategy 1"
+      if vars(self).get("default", None) is None:
+        opts += " -b_strategy 1"
```

Comment:
Hmmm... introducing a workaround for another workaround.  Clearly we can take a better approach than this.  Also, a variable called `default` is a bit too broad in meaning.

Reply:
Can you give me some advice?

Reply:
How about:

```
if get_media()._get_gpu_gen() >= 11 and self.codec.startswith("hevc"):
  if vars(self).get("wa_ldb", True):
    opts += " -b_strategy 1"
```

...then in your custom test config file, add `wa_ldb = False` to the test case definition.